enterprise/web.jsf.editor/src/org/netbeans/modules/web/jsf/editor/index/CompositeComponentModel.java

```diff
@@ -302,9 +303,9 @@ public static class Factory extends JsfPageModelFactory {
 
         @Override
         public JsfPageModel getModel(HtmlParserResult result) {
-            Node node = result.root(LibraryUtils.COMPOSITE_LIBRARY_NS);
+            Node node = result.root(LibraryUtils.COMPOSITE_LIBRARY_JCP_NS);
             if (node == null || node.children().isEmpty()) {
-                node = result.root(LibraryUtils.COMPOSITE_LIBRARY_LEGACY_NS);
+                node = result.root(LibraryUtils.COMPOSITE_LIBRARY_SUN_NS);
```

Comment:
This looks like the indexing for composite components. And my gut feeling is, that without adding the new jakarta namespace (a for-look over all three options seems sane, breaking once one matches.

Reply:
Preparing a fix. I guess there are quite more parts where the jakarta namespace needs to be added. This especially seems to be the case for templates.